controller/network/network.go

```diff
@@ -330,6 +330,11 @@ func (network *Network) LinkConnected(id string, connected bool) error {
 
 	if l, found := network.linkController.get(id); found {
 		if connected {
+			if state := l.CurrentState(); state != nil && state.Mode != Pending {
+				log.Infof("link is %v not pending, cannot mark connected", state.Mode)
```

Comment:
maybe "link **state** is %v..."? making it a bit more clear in the log?

Reply:
fixed